projects/plugins/jetpack/class.json-api-endpoints.php

```diff
@@ -1353,6 +1353,7 @@ function get_media_item_v1_1( $media_id, $media_item = null, $file = null ) {
 			'description' => $media_item->post_content,
 			'alt'         => get_post_meta( $media_item->ID, '_wp_attachment_image_alt', true ),
 			'icon'        => wp_mime_type_icon( $media_item->ID ),
+			'size'        => size_format( filesize( get_attached_file( $media_item->ID ) ), 2 ),
```

Comment:
Nice! This tests well for me. One thing we could do to clean this up is to use `$attachment_file` instead, since that's declared on line 1335 with `get_attached_file` already.

Reply:
Oh, nice spot! Thank you - fixed! :)